### PR TITLE
[Improvement](external table) support hive external table which stores data on tencent chdfs

### PR DESCRIPTION
**Pull request by Yulei-Yang**
Some hive users store hive table in tencent chdfs, currently doris cannot read such tables. this pr enable it in external table scenario.

# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [x] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [x] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by morningman** [comment]
Could you please support this feature in multi-catalog.
The hive external table will be deprecated in future

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 35.03 seconds
 load time: 629 seconds
 storage size: 17122942615 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221216061323_clickbench_pr_64299.html

**Comment by Yulei-Yang** [comment]
> Could you please support this feature in multi-catalog. The hive external table will be deprecated in future

sure, this is the next plan. I will submit an another pr to do this.

File: fe/fe-core/src/main/java/org/apache/doris/catalog/HiveMetaStoreClientHelper.java [hunk 1]
```diff
@@ -761,7 +761,8 @@ public static String showCreateTable(org.apache.hadoop.hive.metastore.api.Table
             if (remoteTable.getPartitionKeys().size() > 0) {
                 output.append("PARTITIONED BY (\n")
                         .append(remoteTable.getPartitionKeys().stream().map(
-                                partition -> String.format("  `%s` `%s`", partition.getName(), partition.getType()))
+                                        partition ->
+                                                String.format(" `%s` `%s`", partition.getName(), partition.getType()))
```
Comment (lide-reed) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/HiveMetaStoreClientHelper.java:765]: Is it need to change to such format?

**Review by lide-reed — APPROVED** [review]
LGTM

**Pull request merged**